packages/react-charting/src/utilities/SVGTooltipText.tsx

```diff
@@ -41,6 +41,16 @@ interface ISVGTooltipTextProps {
    */
   maxHeight?: number;
 
+  /**
+   * Pass false to make the data not focusable, though keyboard. true by default
+   */
+  shouldReceiveFocus?: boolean;
+
+  /**
+   * Pass true to show tooltip directly, false by default
+   */
+  showTooltip?: boolean;
```

Comment:
isn't showTooltip same as isTooltipVisible

Reply:
Yes they're same, but this is a prop, while isTooltipVisible is a state. 

In the case of pie chart it was required to show the tooltip on arc focus.

Reply:
Yes it's the doing the same thing, its a prop while the isTooltipvisible is a state. I'll rename the variable, update the comment to make it more clear.

The requirement was to have focus on the tooptip, when the arc is focused. Should I look for an alternate solution to this approach?